wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/scm/AuthorizingFileContentProvider.java

```diff
@@ -140,7 +147,14 @@ protected String formatUrl(String fileURL) throws DevfileException {
     return requestURL;
   }
 
-  protected String formatAuthorization(String token) {
+  /**
+   * Formats authorization header value.
+   *
+   * @param token personal access token
+   * @param isPAT true if token is personal access token, false if it is OAuth token
+   * @return formatted authorization header value
+   */
+  protected String formatAuthorization(String token, boolean isPAT) {
```

Comment:
We are going to have a conflict when https://github.com/eclipse-che/che-server/pull/451 is merged. I prefer to have a separate method for `Basic` authentication.